checker/build.xml

```diff
@@ -654,12 +654,12 @@
     </target>
 
     <target name="nullness-tests"
-            depends="jar,build-tests,nullness-base-tests,nullness-base-tests-with-asserts,nullness-concurrent-tests,nullness-skipuses-tests,nullness-skipdefs-tests,nullness-uninit-tests,nullness-uninit-tests-with-asserts,nullness-assume-assertions-are-enabled-tests,nullness-extra-tests,nullness-reflection-tests,nullness-invariantarrays-tests,nullness-safedefaults-tests"
+            depends="jar,build-tests,nullness-base-tests,nullness-base-tests-with-asserts,nullness-concurrent-tests,nullness-skipuses-tests,nullness-skipdefs-tests,nullness-uninit-tests,nullness-uninit-tests-with-asserts,nullness-assume-assertions-are-enabled-tests,nullness-extra-tests,nullness-reflection-tests,nullness-invariantarrays-tests,nullness-safedefaultsbytecode-tests,nullness-safedefaultssourcecode-tests"
```

Comment:
This is unrelated to your change, but we should change this: at the moment this list of dependencies is duplicated, which is bad. Moreover, this target doesn't work. In the "checker" directory, do "ant clean; ant nullness-tests". I would expect that to work and that the jdk.jar is built. However, the test case fails.

Should this simply be "depends="jdk.jar,nullness-tests-nojdk"?